tests/test_runs.py

```diff
@@ -212,3 +214,113 @@ def test_create_run_duplicate_tag_key():
     )
     assert result.exit_code != 0
     assert "Duplicate tag key" in result.output
+
+
+def test_link_traces_single_trace():
+    mock_client = MagicMock()
+    with patch("mlflow.runs.MlflowClient", return_value=mock_client):
+        result = CliRunner().invoke(
+            link_traces,
+            ["--run-id", "test_run_123", "--trace-id", "trace_abc"],
+        )
+
+        assert result.exit_code == 0
+        assert "Successfully linked 1 trace(s) to run 'test_run_123'" in result.output
+        mock_client.link_traces_to_run.assert_called_once_with(["trace_abc"], "test_run_123")
+
+
+def test_link_traces_multiple_traces():
+    mock_client = MagicMock()
+    with patch("mlflow.runs.MlflowClient", return_value=mock_client):
+        result = CliRunner().invoke(
+            link_traces,
+            [
+                "--run-id",
+                "test_run_456",
+                "--trace-id",
+                "trace_1",
+                "--trace-id",
+                "trace_2",
+                "--trace-id",
+                "trace_3",
+            ],
+        )
+
+        assert result.exit_code == 0
+        assert "Successfully linked 3 trace(s) to run 'test_run_456'" in result.output
+        mock_client.link_traces_to_run.assert_called_once_with(
+            ["trace_1", "trace_2", "trace_3"], "test_run_456"
+        )
+
+
+def test_link_traces_with_short_option():
+    mock_client = MagicMock()
+    with patch("mlflow.runs.MlflowClient", return_value=mock_client):
+        result = CliRunner().invoke(
+            link_traces,
+            ["--run-id", "run_789", "-t", "trace_x", "-t", "trace_y"],
+        )
+
+        assert result.exit_code == 0
+        assert "Successfully linked 2 trace(s) to run 'run_789'" in result.output
+        mock_client.link_traces_to_run.assert_called_once_with(["trace_x", "trace_y"], "run_789")
+
+
+def test_link_traces_file_store_error():
+    mock_client = MagicMock()
+    mock_client.link_traces_to_run.side_effect = MlflowException(
+        "Linking traces to runs is not supported in FileStore. "
+        "Please use a database-backed store (e.g., SQLAlchemy store) for this feature."
+    )
+    with patch("mlflow.runs.MlflowClient", return_value=mock_client):
```

Comment:
```suggestion
    with patch("mlflow.runs.MlflowClient.link_traces_to_run", side_effect = MlflowException(
        "Linking traces to runs is not supported in FileStore. "
        "Please use a database-backed store (e.g., SQLAlchemy store) for this feature."
    )):
```

Reply:
can you apply this change to other `mock.patch`?

Reply:
Done